packages/admin-ui/src/table/hooks/use-vertical-scroll.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react'
+import { useStateContext } from '../context'
+
+export function useVerticalScroll() {
```

Comment:
same for the file name

```suggestion
export function useTableScroll() {
```